occupi-backend/configs/config.go

```diff
@@ -239,3 +241,12 @@ func GetOTPExpiration() int {
 	}
 	return expiration
 }
+
+// gets the frontend url as defined in the config.yaml file in seconds
+func GetFrontendURL() string {
+	frontendURL := viper.GetString("FRONTEND_URL")
```

Comment:
For this, it is better to use the "FrontendURL" you defined on line 32 instead of an actual string

Reply:
alright will change them